### PR TITLE
chore(release): v0.12 release notes

### PR DESCRIPTION
**Pull request by wass3r**

### CODE REVIEW

**Review by KellyMerrick — APPROVED** [review]
lgtm

**Comment by wass3r** [comment]
sorry - left out UI changes by accident 😲

File: releases/v0.12.md [hunk 1]
```diff
@@ -0,0 +1,74 @@
...
+* (server) feat(repo)!: add build rate limiting by @jbrockopp in https://github.com/go-vela/server/pull/558
+  - introduces new behavior
+* (server) feat(repo/webhook)!: handle repository rename in database by @ecrupper in https://github.com/go-vela/server/pull/556
+  - depends on a new DB field
+* (server) feat(secrets)!: adding new metadata to secrets by @ecrupper in https://github.com/go-vela/server/pull/526
+  - depends on new DB fields
+
+### Bug Fixes
+
+* (cli) fix(validate): handle .yaml default file by @wass3r in https://github.com/go-vela/cli/pull/326
```
Comment (kneal) [inline releases/v0.12.md:37]: Just a formatting comment, I personally like what was done [here](https://github.com/go-vela/community/blob/fde93e0c039dbb923949621fb4d6c4a43b569e2f/releases/v0.11.md#bug-fixes-1) vs the raw link.

Comment (wass3r) [reply releases/v0.12.md:37]: fair.. i pulled these directly from the GitHub generated release notes, hence the difference. I will adjust to adapt the previous format for this. incoming..

Comment (kneal) [reply releases/v0.12.md:37]: Really liked the `@<user>` for call outs! I remember us trying to think of ways of doing that in the past.

Comment (wass3r) [reply releases/v0.12.md]: made the PR reference edit

Comment (wass3r) [reply releases/v0.12.md]: @kneal yes, i liked that as well.. i think where it gets a bit tricky is that to be inclusive we are leaving out folks that contribute in other ways. whether that's bug reports, feature requests, reviewing, discussion participants, etc. - kind of in the spirit of https://github.com/all-contributors. i think the current form of this presented here is not final form.. maybe it'll be closer to `all-contributors`?

Comment (kneal) [reply releases/v0.12.md]: Yeah, that's true. 😞 

There's definitely a lot more to a feature than just being the code author.

File: releases/v0.12.md [hunk 2]
```diff
@@ -0,0 +1,74 @@
...
+## v0.12.0
+
+Happy February Vela Community!
+
+Below is a summary of what's new in 0.12.0
+
+### Breaking Changes
+
+* (types) fix(secret)!: remove implicit repo secrets and clean up secrets testing by @GregoryDosh in [#210](https://github.com/go-vela/types/pull/210)
+  - existing implicit secrets usage will cease to work
```
Comment (JordanSussman) [inline releases/v0.12.md:27]: Can you add some more clarification on the implications of this? Bit unclear to me what this means without reading through the source code in that pull request.

Comment (wass3r) [reply releases/v0.12.md:27]: added a commit to check out - does that help?

Comment (jbrockopp) [reply releases/v0.12.md:27]: I thought this change was reverted and not introduced into `v0.12.x`?

Revert PR - https://github.com/go-vela/types/pull/229

Reimplemented PR - https://github.com/go-vela/types/pull/230

Comment (kneal) [reply releases/v0.12.md:27]: Very late to the party but it probably woulda been nice to add what this [character](https://github.com/go-vela/types/pull/210/files#diff-4df2bf19e26362114b475b25b13f8ffd4b2dc54037729f3f28a7185c92caaabcR25) was, I had to google and do some reading on it. Woulda helped know from the PR link the purpose in the release notes.

CC: @GregoryDosh

Comment (wass3r) [reply releases/v0.12.md:27]: @jbrockopp oh yes.. my fault. i removed some of the PRs and this one slipped through! when i pulled the release notes 😵

Comment (jbrockopp) [reply releases/v0.12.md:27]: It's all good! ..for a moment there I thought I was going crazy 😅 🤪

Comment (wass3r) [reply releases/v0.12.md]: fixed - nothing to see on that topic for this release

**Review by jbrockopp — APPROVED** [review]
LGTM

**Review by JordanSussman — APPROVED** [review]

**Pull request merged**